arm/README.md

```diff
@@ -91,6 +91,58 @@ post,
 [Developer's Guide to Auth with Azure Resource Manager API](http://www.dushyantgill.com/blog/2015/05/23/developers-guide-to-auth-with-azure-resource-manager-api/),
 that is also quite helpful.
 
+### Use an authentication file
```

Comment:
I'm not sure that the AzureRM README file is really the best place for this. Honestly, I think we're overdue to start a Wiki in our repository. We could have a page on authentication, and a page for the roadmap as mentioned in #804.